src/WCSimSteppingAction.cc

```diff
@@ -100,7 +100,7 @@ G4int WCSimSteppingAction::G4ThreeVectorToWireTime(G4ThreeVector *pos3d,
 } 
 
 
-void WCSimSteppingAction::Distortion(G4double x,G4double y)
+void WCSimSteppingAction::Distortion(G4double /*x*/,G4double /*y*/)
```

Comment:
Sorry this was my fault - commiting some stuff to get rid of most of the (numerous) benign compiler warnings - when not quite understanding the github workflow. I think most (if not all) of these types of changes are in 495cb78ff834bf1b65d7cfebde497cb299bdbc85
I think the changes are valuable (so we can see actual compilation problems clearly) but how you want to proceed is up to you - either do it as part of this pull request, or as another one. If the latter, do you know the git voodoo to apply only commit 495cb78ff834bf1b65d7cfebde497cb299bdbc85 to a branch based off develop?